platform/macos/app/Base.lproj/MainMenu.xib

```diff
@@ -135,7 +135,7 @@
                             </menuItem>
                             <menuItem title="Print…" keyEquivalent="p" id="aTl-1u-JFS">
                                 <connections>
-                                    <action selector="print:" target="-1" id="qaZ-4w-aoO"/>
+                                    <action selector="print:" target="Voe-Tx-rLC" id="rXg-Pd-AG8"/>
```

Comment:
This suggests that the map view isn't in the responder chain somehow, which is unexpected. Locally, I hooked the map view up to the window's initialFirstResponder, but that didn't seem to be sufficient. I wonder if the recently added Layers table view is stealing keyboard focus.